### PR TITLE
Feature/cxp 933 releasing doc

### PR DESCRIPTION
**Pull request by yasskin**
## PR Checklist

- [x] PR has one of the `semver-` labels
- [ ] Two core team engineer approvals
- [ ] One core team UX approval

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Before updating the documentation and releasing to npm, follow these steps:
+
+1. `git pull origin master`: Ensure that `master` is clean and in sync with GitHub.
+2. `npm install`: Ensure that you have the latest dependencies.
+3. `npm run test`: Run the test suite--if you need to update the Jest snapshots: `npm run snapshot`.
+4. `npm login`: Ensure that you're logged into npm.
+
+## Update documentation
+
+After enrusing the release is production-ready, determine a suitable version number based on the types of changes in the release.
```
Comment (dreidn) [inline RELEASING.md]: Typo here. Looks like it should be 'ensuring'

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+We follow [semver](https://semver.org/), which presrcribes that:
```
Comment (dreidn) [inline RELEASING.md]: another typo, should be 'prescribes'

File: RELEASING.md (hunk 1 above)
Comment (yasskin) [reply RELEASING.md]: Good eye! Corrected.

File: RELEASING.md (hunk 2 above)
Comment (yasskin) [reply RELEASING.md]: Good eye! Corrected.

**Pull request merged**